src/storage/upgrade_project.ts

```diff
@@ -28,37 +28,44 @@ import * as storageModule from './module';
 import * as storageModuleContent from './module_content';
 import * as storageNames from './names';
 import * as storageProject from './project';
+import { ClassMethodDefBlock, Parameter, BLOCK_NAME as MRC_CLASS_METHOD_DEF_BLOCK_NAME } from '../blocks/mrc_class_method_def';
 
+export const NO_VERSION = '0.0.0';
+export const CURRENT_VERSION = '0.0.3';
 
 export async function upgradeProjectIfNecessary(
-    storage: commonStorage.Storage, projectName: string): Promise<void> {
+  storage: commonStorage.Storage, projectName: string): Promise<void> {
```

Comment:
Continued lines should be indented 4 spaces.

Reply:
done